common_helper_yara/yara_scan.py

```diff
@@ -45,12 +57,12 @@ def _parse_yara_output(output):
 
 
 def _split_output_in_rules_and_matches(output):
-    split_regex = re.compile(r'\n*.*\[.*\]\s\/.+\n*')
+    split_regex = re.compile(r'\n*.*\[.*]\s/.+\n*')
```

Comment:
We should rewrite this expression for reasonable reasons.

Reply:
Seems to work as it is. I only removed unnecessary escapes